tests/datasets/test_views.py

```diff
@@ -4251,3 +4257,64 @@ def test_dataset_rdf_download__datas_service(app: DjangoTestApp):
     </dcat:DataService>
 </rdf:RDF>"""
     )
+
+
+class TestDatasetDeleteView:
+    def test_delete_dataset(self, app: DjangoTestApp) -> None:
+        user = UserFactory(is_staff=True)
+        app.set_user(user)
+
+        dataset = DatasetFactory()
+
+        app.post(reverse("dataset-delete", args=[dataset.pk]))
+        assert not Dataset.objects.exists()
```

Comment:
Please assert using PK, as this could lead to test Fail with https://github.com/atviriduomenys/katalogas/pull/1841 🙂 


```suggestion
        app.post(reverse("dataset-delete", args=[dataset.pk]))
        assert not Dataset.objects.filter(pk=dataset.pk).exists()
```

Reply:
Fully agree. Fixed in 797ada5082a2f6ca2ed2730bd11268a8a28e03cc